pkgs/applications/misc/hunter/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, callPackage
+, makeRustPlatform
+, fetchFromGitHub
+, IOKit ? null
+, makeWrapper
+, glib
+, gst_all_1
+, libsixel
+}:
+
+assert stdenv.isDarwin -> IOKit != null;
+
+let
+  date = "2020-05-22";
```

Comment:
Is the current rust version in nix now sufficient?

Reply:
Nothing changed upstream since May 25.
Shall I mark this PR as WIP until [upstream doesn't need nightly](https://github.com/rabite0/hunter/issues/89) anymore?

Reply:
Yes please. I don't think it is a good idea to include the overlay in nix.